docker/docker.engine.yaml

```diff
@@ -0,0 +1,152 @@
+telemetry:
+  environment: local
+  version: ${env:VERSION}
+  logging:
+    level: debug # debug, info, warn, error
+    encoding: console
+
+api:
+  development: true
+  host: 0.0.0.0
+  port: 9099
+  analytics:
+    enabled: false
+    # Uncomment to customize analytics
+    # max_buffer_size_mb: 10
+    # flush_size_mb: 0.5
+    # flush_interval_sec: 30
+    # target_url: "https://cloud.arcade-ai.com/analytics"
+  # Uncomment to enable rate limiter
+  # rate_limit:
+  #   redis:
+  #     addr: "localhost:6379"
+  #     password: ""
+  #     db: 2
+  #     time_unit: m
+  #     limit: 2
+  #     write_timeout: 10
+  #     read_timeout: 10
+
+llm:
+  models:
+    - id: oai
+      openai:
+        api_key: ${env:OPENAI_API_KEY}
+    # - id: anthropic
+    #   anthropic:
+    #     api_key: ${env:ANTHROPIC_API_KEY}
+    #     #    model: "claude-3-5-sonnet-20240620"
+    #- id: ollama
+    #  openai:
+    #    base_url: "http://localhost:11434/v1"
+    #    api_key: "ollama"
+
+tools:
+  directors:
+    - id: default
+      enabled: true
+      workers:
+        - id: "localworker"
+          enabled: true
+          http:
+            uri: ${env:WORKER_URL}
+            timeout: 30
+            retry: 3
+            secret: ${env:ARCADE_WORKER_SECRET} # If not set, defaults to "dev" in development mode only
+          # Uncomment mock and comment http to start engine without live worker
+          # mock:
+          #   enabled: true
+
+security:
+  root_keys:
+    - id: key1
+      default: true
+      value: ${env:ROOT_KEY_1}
+    # - id: key2
+    #   value: ${env:ROOT_KEY_2}
+
+storage:
+  # postgres:
+  #   user: postgres
+  #   password: 123456
+  #   host: localhost
+  #   port: 5432
+  #   db: arcade_engine
+  #   sslmode: disable
+  sqlite: # Default for local development
+    connection_string: "/app/arcade-engine.sqlite3"
+
+cache:
+  api_key_ttl: "10s"
+  redis:
+    addr: ${env:REDIS_HOST}:${env:REDIS_PORT}
+    password: ${env:REDIS_PASSWORD}
+
+auth:
+  token_store:
+    redis:
+      addr: ${env:REDIS_HOST}:${env:REDIS_PORT}
+      password: ${env:REDIS_PASSWORD}
+
+  providers:
```

Comment:
I will need to edit this structure once the new provider stuff goes in and/or gets released. Not sure who will merge first.

Reply:
@wdawson can i follow the changes you made in the engine repo for this?

Reply:
I believe I made those changes in this PR. Yes, what's there now should be good!